src/main/java/mousio/etcd4j/transport/EtcdNettyClient.java

```diff
@@ -392,7 +413,6 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws E
   @Override
   public void close() {
     logger.info("Shutting down Etcd4j Netty client");
-    this.timer.stop();
```

Comment:
RetryWithExponentialBackOff.maxDelay default is -1, and whenever retry, delay *= 2, i think this is not a goods solution, because the delay may be very large, and when died etcd come back, this retry will not work.